src/main/java/com/es/phoneshop/model/product/SortField.java

```diff
@@ -0,0 +1,6 @@
+package com.es.phoneshop.model.product;
+
+public enum SortField {
+    price,
```

Comment:
Enum values are named like constants: uppercase with underscores (for multiword ones).